README.md

```diff
@@ -137,6 +137,12 @@ webshop/
    ├─ nfk.json
 ```
 
+## Release
+
+The webshop is automatically released to staging then new changes on the master branch. To release to production edit the latest release version in GitHub and select _Publish release_.
+
+Normally we increment the minor version when releasing a new version, but if there are no visible changes and only small fixes then the patch version may be incremented.
```

Comment:
```suggestion
Normally we increment the minor version when releasing a new version, but if there are no visible changes and only small fixes then the patch version may be incremented. Bump as major version if there are any changes that require the release of webshop-backend or any other dependency.
```

Reply:
> Tenker det kunne vært interessant å f.eks si at major change = avhengig av eksterne releases (atb-ticket, webshop-backend, etc).

Kunne vi spesifisert enda litt tydeligere når det er nødvendig med major og ikke, for det er rimelig uklart for meg. Jeg har i hovedsak tenkt på major versjon som større rebrandings eller omskriving av app. Er det ny major hvis man kaller et nytt endepunkt i en av intergrasjonene? Hva hvis man lytter på en ny collection i Firestore? Hva hvis man krever RemoteConfig-verdier i ny struktur?

Reply:
Major, minor og patch handler i utgangspunktet om API-overflate i ren semantisk versjonering. Det gir ikke nødvendigvis så mye mening i sluttprodukter som dette. Derfor tenkte jeg det kunne vært interessant å heller snu på det litt slik at vi enklere kan fange opp behov til eksterne. Så fremfor at major (breaking change) indikerer at det er endring i produktet her så er det endring i krav. Altså _avhengig av eksterne avhengigheter som har hatt breaking change_.

Det er vanskelig å følge med i release løpet om det er avhengig av endringer. Kanskje må det rulles ut endringer i Firebase Store? Kanskje må det rulles ut endringer på webshop-backend? Kanskje atb-ticket? Eller faktisk abt-cloud-function som har ny reserve change handler?

Vi husker det i staging ved merge, men enkelt å glemme når det er prodsetting.

Dersom slike endringer blir gjort i frontend (når de kodes) og markeres med semantiske commits som f.eks

```
breaking-change: updates reserve handling to use Firebase Store

Requires Cloud Function update-reservation
```

Så synliggjøres det i release notes i generering av det og derfor ved publish av release så blir vi klar over det.

Reply:
Eksempel: https://github.com/AtB-AS/webshop/pull/487